test/Microsoft.Azure.WebJobs.Host.FunctionalTests/Indexers/FunctionIndexerTests.cs

```diff
@@ -348,8 +352,11 @@ public void CheckRetrySupport_LeavesRetry()
 
             // Act & Assert
             FunctionIndexer.CheckRetrySupport(new SupportRetryTestTriggerBinding(), desc, logger);
-            Assert.NotNull(desc.RetryStrategy);
-            Assert.DoesNotContain(loggerProvider.GetAllLogMessages(), x => x.FormattedMessage.Contains("Retries are not supported by the trigger binding for "));
+            Assert.Contains(loggerProvider.GetAllLogMessages(), x => x.FormattedMessage.Contains("Retry strategy is defined for function ") && x.Level == LogLevel.Debug);
+
+            //Temporary comment: https://github.com/Azure/azure-webjobs-sdk/issues/2788
+            //Assert.NotNull(desc.RetryStrategy);
```

Comment:
delete commented code?

Reply:
no, this code will be uncommented once https://github.com/Azure/azure-webjobs-sdk/issues/2788 is resolved